### PR TITLE
first pass at replicating chem predicate sheet fixes #662 fixes #85 fixes #332

### LINKED ISSUE #85: Utility of mixin slots for 'stimulates' and 'inhibits?
Thoughts on mixin predicates for the notions of 'stimulates' and 'inhibits' - which are broader than the existing notions of positively and negatively regulates.  These could live under blm:affeccts, and be mixin parents of blm:positively regulates and blm:negatively regulates, respectively.

One use case here is that SemMedDB uses STIMULATES and INHIBITS predicates largely to describe regulation between entities (e.g. gene/protein regulating gene/protein), but also in other ways that wouldn't qualify as entity-entity regulation (e.g. statements that a procedure STIMULATES physiologic functions, and statements that a chemical INHIBITS cellular functions).

Beyond this specific use case (which may not be long lived if we abandon semmeddb as a reliable data KS for the translator) would it  generally be useful to have predicates  for describing cases where thing 1 stimulates/increases or impairs/decreases thing 2 (which may or may not cases of  biological regulation)?

### LINKED ISSUE #332: Hypothetical modeling of specific pharmacological modes of action
Naive question here...  In the context of Translator, it seems likely that we will eventually want to model some detailed pharmacological modes of action, e.g., "antagonist" vs "inverse agonist".  As a strawman to better understand BioLink Model (and in advance of the formal discussion/approval of these decisions), can someone walk me through the way this _would_ be modeled?

I see an existing slot for `negatively regulates, entity to entity`:
```
  negatively regulates, entity to entity:
    aliases: ['activity directly negatively regulates activity of']
    is_a: regulates, entity to entity
    mixins:
      - negatively regulates
    local_names:
      translator: negatively regulates
      ro: activity directly negatively regulates activity of
    in_subset:
      - translator_minimal
    slot_uri: RO:0002449
    mappings:
      - SEMMEDDB:INHIBITS
```
So would antagonism and inverse agonism be modeled as another subclass slot? Would they be direct subclasses of `negatively regulates, entity to entity`, or would you create some intermediate class like `negatively regulates, chemical substance to gene or gene product`? Am I in the right ballpark here?

### LINKED ISSUE #662: DRKG predicate mappings
As part of a research project, I am using DRKG (The Drug Repurposing Knowledge Graph)  and am hoping to map it to biolink so that I can do some analysis comparing DRKG and ROBOKOP. 

Would it be possible to add predicate mappings for the relations in DRKG that are not already mapped? They are listed below.

- DRUGBANK::carrier::Compound:Gene
- DRUGBANK::ddi-interactor-in::Compound:Compound
- DRUGBANK::enzyme::Compound:Gene
- DRUGBANK::target::Compound:Gene
- DRUGBANK::x-atc::Compound:Atc
- DRUGBANK::treats::Compound:Disease
- GNBR::A+::Compound:Gene
- GNBR::A-::Compound:Gene
- GNBR::B::Compound:Gene
- GNBR::B::Gene:Gene
- GNBR::C::Compound:Disease
- GNBR::D::Gene:Disease
- GNBR::E+::Compound:Gene
- GNBR::E+::Gene:Gene
- GNBR::E-::Compound:Gene
- GNBR::E::Compound:Gene
- GNBR::E::Gene:Gene
- GNBR::G::Gene:Disease
- GNBR::H::Gene:Gene
- GNBR::I::Gene:Gene
- GNBR::J::Compound:Disease
- GNBR::J::Gene:Disease
- GNBR::K::Compound:Gene
- GNBR::L::Gene:Disease
- GNBR::Md::Gene:Disease
- GNBR::Mp::Compound:Disease
- GNBR::N::Compound:Gene
- GNBR::O::Compound:Gene
- GNBR::Pa::Compound:Disease
- GNBR::Pr::Compound:Disease
- GNBR::Q::Gene:Gene
- GNBR::Rg::Gene:Gene
- GNBR::Sa::Compound:Disease
- GNBR::T::Compound:Disease
- GNBR::Te::Gene:Disease
- GNBR::U::Gene:Disease
- GNBR::Ud::Gene:Disease
- GNBR::V+::Gene:Gene
- GNBR::W::Gene:Gene
- GNBR::X::Gene:Disease
- GNBR::Y::Gene:Disease
- GNBR::Z::Compound:Gene
- GNBR::in_tax::Gene:Tax
- Hetionet::AdG::Anatomy:Gene
- Hetionet::AeG::Anatomy:Gene
- Hetionet::AuG::Anatomy:Gene
- Hetionet::CbG::Compound:Gene
- Hetionet::CcSE::Compound:Side Effect
- Hetionet::CdG::Compound:Gene
- Hetionet::CpD::Compound:Disease
- Hetionet::CrC::Compound:Compound
- Hetionet::CtD::Compound:Disease
- Hetionet::CuG::Compound:Gene
- Hetionet::DaG::Disease:Gene
- Hetionet::DdG::Disease:Gene
- Hetionet::DlA::Disease:Anatomy
- Hetionet::DpS::Disease:Symptom
- Hetionet::DrD::Disease:Disease
- Hetionet::DuG::Disease:Gene
- Hetionet::GcG::Gene:Gene
- Hetionet::GiG::Gene:Gene
- Hetionet::GpBP::Gene:Biological Process
- Hetionet::GpCC::Gene:Cellular Component
- Hetionet::GpMF::Gene:Molecular Function
- Hetionet::GpPW::Gene:Pathway
- Hetionet::Gr>G::Gene:Gene
- Hetionet::PCiC::Pharmacologic   Class:Compound
- INTACT::ADP RIBOSYLATION   REACTION::Gene:Gene
- INTACT::ASSOCIATION::Compound:Gene
- INTACT::ASSOCIATION::Gene:Gene
- INTACT::CLEAVAGE REACTION::Gene:Gene
- INTACT::COLOCALIZATION::Gene:Gene
- INTACT::DEPHOSPHORYLATION   REACTION::Gene:Gene
- INTACT::DIRECT INTERACTION::Compound:Gene
- INTACT::DIRECT INTERACTION::Gene:Gene
- INTACT::PHOSPHORYLATION   REACTION::Gene:Gene
- INTACT::PHYSICAL ASSOCIATION::Compound:Gene
- INTACT::PHYSICAL ASSOCIATION::Gene:Gene
- INTACT::PROTEIN CLEAVAGE::Gene:Gene
- INTACT::UBIQUITINATION REACTION::Gene:Gene
- STRING::ACTIVATION::Gene:Gene
- STRING::BINDING::Gene:Gene
- STRING::CATALYSIS::Gene:Gene
- STRING::EXPRESSION::Gene:Gene
- STRING::INHIBITION::Gene:Gene
- STRING::OTHER::Gene:Gene
- STRING::PTMOD::Gene:Gene
- STRING::REACTION::Gene:Gene
- bioarx::Coronavirus_ass_host_gene::Disease:Gene
- bioarx::Covid2_acc_host_gene::Disease:Gene
- bioarx::DrugHumGen:Compound:Gene
- bioarx::DrugVirGen:Compound:Gene
- bioarx::HumGenHumGen:Gene:Gene
- bioarx::VirGenHumGen:Gene:Gene

@balhoff

### PR DESCRIPTION
**Pull request by sierra-moxon**

### CODE REVIEW

**Comment by nlharris** [comment]
Chemical predicate spreadsheet: https://docs.google.com/spreadsheets/d/1VI2s5YVDwOHiQv7hBWkl12d86yWvm-z_pDJFnDwsoiY/edit#gid=548945777

**Comment by nlharris** [comment]
Requesting review from @schatzkar and team (see #662)

**Comment by nlharris** [comment]
This PR still seems to be in limbo

File: biolink-model.yaml [hunk 1]
```diff
@@ -5017,17 +5184,36 @@ slots:
...
+      - GNBR:substrate
 
   is substrate of:
-    is_a: participates in
+    is_a: decreases molecular modification of
     inverse: has substrate
 
+  has product:
+    description: >-
+      The end result of a manufacturing process; anything that is produced.
```
Comment (mbrush) [inline biolink-model.yaml:5210]: Definition here seems at odds with application to molecular products of catalyzed reactions

**Pull request closed without merging**